src/main/resources/data-use-v3.json

```diff
@@ -0,0 +1,83 @@
+{
+  "id": "https://consent-ontology.dsde-prod.broadinstitute.org/schemas/data-use",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Data Use Schema V3",
+  "version": 2,
+  "type": "object",
+  "dependencies": {
+    "recontactingDataSubjects": {
+      "oneOf": [
+        {"required": ["recontactMay"]},
+        {"required": ["recontactMust"]}
+      ]
+    },
```

Comment:
Don't need this, AFAICT - it's not in https://github.com/DataBiosphere/consent/blob/develop/src/main/resources/dataset-registration-schema_v1.json